eth/rpc/backend/utils.go

```diff
@@ -212,7 +214,7 @@ func (b *Backend) retrieveEVMTxFeesFromBlock(
 func AllTxLogsFromEvents(events []abci.Event) ([][]*gethcore.Log, error) {
 	allLogs := make([][]*gethcore.Log, 0, 4)
 	for _, event := range events {
-		if event.Type != evm.EventTypeTxLog {
+		if event.Type != proto.MessageName((*evm.EventTxLog)(nil)) {
```

Comment:
```suggestion
		if event.Type != proto.MessageName(new(evm.EventTxLog)) {
```

Technically equivalent, but nil pointers look a bit concerning